sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -966,7 +966,7 @@ class SessionCatalog(
   }
 
   def isGlobalTempViewDB(dbName: String): Boolean = {
-    globalTempViewManager.database.equals(dbName)
+    globalTempViewManager.database.equalsIgnoreCase(dbName)
```

Comment:
This is a follow up change to https://github.com/apache/spark/pull/36586/files/2864b6ab46cde170c7b104d572b265cbc1e13687#r893029290